apps/testing/src/main/java/com/akto/test_editor/strategy/Strategy.java

```diff
@@ -0,0 +1,70 @@
+package com.akto.test_editor.strategy;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+
+import com.akto.dao.ApiCollectionsDao;
+import com.akto.dao.context.Context;
+import com.akto.dto.ApiCollection;
+import com.akto.dto.testing.GenericTestResult;
+import com.akto.dto.testing.TestResult;
+import com.akto.dto.testing.YamlTestResult;
+import com.akto.log.LoggerMaker;
+import com.akto.log.LoggerMaker.LogDb;
+import com.akto.test_editor.Utils;
+
+public class Strategy {
+    
+    private static final String AKTO_DISCOVERED_APIS_COLLECTION = "shadow_apis";
+    private static final int AKTO_DISCOVERED_APIS_COLLECTION_ID = 1333333333;
+    private static final LoggerMaker loggerMaker = new LoggerMaker(Strategy.class);
```

Comment:
use this prototype instead, https://github.com/akto-api-security/akto/blob/master/libs/utils/src/main/java/com/akto/log/LoggerMaker.java#L73 the one being used is depracted.